tools/generators/legacy/src/DTO/XcodeScheme.swift

```diff
@@ -36,15 +36,22 @@ extension XcodeScheme {
     }
 
     /// Create a new scheme applying any default actions based upon the current scheme.
-    var withDefaults: XcodeScheme {
-        get throws {
-            var buildTargets = [BazelLabel: XcodeScheme.BuildTarget]()
-
-            func enableBuildForValue(
-                _ label: BazelLabel,
-                _ keyPath: WritableKeyPath<XcodeScheme.BuildFor, XcodeScheme.BuildFor.Value>,
-                _ mode: EnableBuildForValueMode = .merge
-            ) throws {
+    func withDefaults(
+        aliasLabels: [BazelLabel: [BazelLabel]] = [:]
```

Comment:
Making this a `func` that can take `aliasLabels` was the easiest way I've found to expand labels in a single place so the scheme action creations that happen next work as expected